src/plugins/note/note.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'jsx-dom';
+
+import {listen, memoize} from '@exadel/esl/modules/esl-utils/decorators';
+import {UIPPlugin} from '../../core/base/plugin';
+
+export class UIPNote extends UIPPlugin {
+  static is = 'uip-note';
+  static title = 'Note! ';
+
+  @memoize()
+  protected get $inner(): HTMLElement {
+    return (<div class={UIPNote.is + '-inner uip-plugin-inner uip-plugin-inner-bg'}></div>) as HTMLElement;
```

Comment:
```suggestion
    return (<div className={`${UIPNote.is}-inner uip-plugin-inner uip-plugin-inner-bg`}></div>) as HTMLElement;
```

Reply:
According to PReact semantic (which is applicable to jsx-dom) both `class` and `className` can be used

@yadamskaya, as we have partially applied `class` and partially `className` on the project, it's up to you to finally decide which one is gonna be used then we can consider adding a rule to our ESLint Config